### ISSUE TITLE
list tabs in current window only?

### ISSUE THREAD

**Issue by craig-sanders** [lior99/see-all-tabs#2]
This seems the best of all the tab-listing extensions i've looked at so far, simple and fast.  and one of the few that's open soure with a github repo.  Thanks for writing it and making it available.

It's **good**.  Very good.  But it would be **great** if it could be made to list tabs in the current window only.   Like _All Tabs Helper_ in Firefox https://addons.mozilla.org/en-US/firefox/addon/all-tabs-helper/

e.g. I have 33 windows open at the moment, with dozens of tabs in each.  When I want to see the list of tabs, I don't want to have to scroll down to hunt for the current window every time I click on a tab or click on the **x** button to close it.

In fact, I don't care at all about tabs in other windows, a list of hundreds of tabs is useless to me.  I "organise" my browsing by windows (opening up a new window for a new topic of interest) and can't recall ever wanting to see a list of tabs in **other** windows - if I'm looking at a tab list, it's to decide whether to 1. go back to that tab or 2. to close it.  Usually I'm looking for tabs to close to reclaim RAM from Chromium.

**tl;dr:** There should be an option to show only tabs in the current window.  Or, at the very least, sort the current window to the top.

**Comment by lior99** [comment]
Hi Craig,
Thank you for your review and comments. I have updated the extension to allow (via user options for the extension) display of only current window, (it can be toggle to on/off) and the current window will be titled as This window and will always be on top for easier access. Also, there is an experimental support for dark theme mode and some UI improvements.

It will take some days for the Chrome Team to review this new version, and then it will be automatically published and updated on users browser.

New version number will be 3.1.0 :)

**Comment by craig-sanders** [comment]
cool, thanks for updating this so quickly.  the updated version works nicely :)
